chapters/ch06/3.spell-correction.ipynb

```diff
@@ -507,22 +500,29 @@
        "      <td>3239</td>\n",
        "      <td>1</td>\n",
        "    </tr>\n",
+       "    <tr>\n",
+       "      <th>84</th>\n",
+       "      <td>phono</td>\n",
+       "      <td>phone</td>\n",
+       "      <td>5</td>\n",
+       "      <td>3065</td>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
        "    misspelling correction  misspell_counts  correction_counts  edit_dist\n",
        "50      iphone3     iphone                6              16854          1\n",
-       "61      laptopa     laptop                6              14119          1\n",
        "62        latop     laptop                5              14119          1\n",
        "136     toucpad   touchpad                6              11550          1\n",
        "137    touxhpad   touchpad                5              11550          1\n",
        "148     wirless   wireless                6              10060          1\n",
-       "127     tableta     tablet                6               8260          1\n",
        "8          cage       case                6               7541          1\n",
        "10         cape       case                5               7541          1\n",
        "30      gallaxy     galaxy                6               5839          1\n",
+       "61      laptopa    laptops                6               5565          1\n",
```

Comment:
This looks very different than what was there before (5565 vs 14119). 

Also lost "tableta" in the process...

I wonder why?

Reply:
I'm not sure why your results are changed in this PR, but the main I have locally matches main on remote.

Reply:
It seems like before the algorithm was determining that "laptopa" was a mispelling of "laptop", but now "laptopa" is a mispelling of "laptops", still investigating